Sources/WebSocket/WebSocket.swift

```diff
@@ -71,6 +71,9 @@ public final class WebSocket: BasicWorker {
 
     /// See `onCloseCode(...)`.
     var onCloseCodeCallback: (WebSocketErrorCode) -> ()
+    
+    /// Can be set to uniquely identify the WebSocket
+    public var id: String?
```

Comment:
Could we make this a computed property based on the `Channel`? Maybe `ObjectIdentifier`?

Reply:
That will work, but then maybe it makes sense to make WebSocket conform to Hashable using the ObjectIdentifier of the Channel. This way we can make Dictionaries like [WebSocket : Value] which is nice for mapping WebSockets to names or other information. I think this might make more sense because you are most likely going to map the ObjectIdentifier to something else anyways.